src/seedu/addressbook/data/person/Email.java

```diff
@@ -58,4 +59,10 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+	@Override
+	public String getPrintableString() {
+		// TODO Auto-generated method stub
```

Comment:
auto generated comments which doesn't add value should be removed. Keep the code clean.